### PR TITLE
[WIP] Create HomePage UI 

### PR DESCRIPTION
**Pull request by duyhung0311**

### CODE REVIEW

File: frontend/src/components/component_ngangxanh/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <div>
+      <div className="box">
+        <div>
+          <div style={{ display: "flex", justifyContent: "center",paddingTop:"10px" }}>
+            <div className="icon-hover">{props.icon}
+              {/* <FontAwesomeIcon
+                icon={faHourglassEnd}
+                size="3x"
+                color="#007c7e"
+              /> */}
```
Comment (hiepvu0201) [inline frontend/src/components/component_ngangxanh/index.js:23]: can this icon be removed?

File: frontend/src/components/menu/LeftMenu.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <Menu.Item key="hoptac">
+        <a href="">Hợp tác</a>
+      </Menu.Item>
+      <Menu.Item key="news">
+        <a href="">Tin tức</a>
+      </Menu.Item>
+      <Menu.Item key="contact">
+        <a href="">Liên hệ</a>
+      </Menu.Item>
+      {/* <SubMenu key="sub1" title={<span>Blogs</span>}> */}
```
Comment (hiepvu0201) [inline frontend/src/components/menu/LeftMenu.js:30]: can this comment be remove?

File: frontend/src/feature/homepage/style.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+.container{
+    width: 80%;
+    height: auto;
+}
+.topic{
+    font-size: 30px;
+    padding-top: 10px;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: bold;
+    /* style={{
```
Comment (hiepvu0201) [inline frontend/src/feature/homepage/style.css:31]: can you remove this comment too?

File: frontend/src/feature/login/style.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    ;
+}
+.form-box{
+    width: 25%;
+    height: auto;
+    display:flex;
+    align-items: center;
+    justify-content: start;
+    border-radius: 10px;
+    /* box-shadow: 0px 0px 10px 2px gray; */
```
Comment (hiepvu0201) [inline frontend/src/feature/login/style.css:17]: can you remove this comment too?

File: frontend/src/components/component_ngangxanh/index.js (hunk 1 above)
Comment (duyhung0311) [reply frontend/src/components/component_ngangxanh/index.js:23]: yes

File: frontend/src/components/menu/LeftMenu.js (hunk 2 above)
Comment (duyhung0311) [reply frontend/src/components/menu/LeftMenu.js:30]: yes

File: frontend/src/feature/homepage/style.css (hunk 3 above)
Comment (duyhung0311) [reply frontend/src/feature/homepage/style.css:31]: yes

File: frontend/src/feature/login/style.css (hunk 4 above)
Comment (duyhung0311) [reply frontend/src/feature/login/style.css:17]: yes

**Pull request merged**